main/media_cache.c

```diff
@@ -333,7 +333,7 @@ int ast_media_cache_create_or_update(const char *uri, const char *file_path,
 	}
 
 	if (created && ast_bucket_file_create(bucket_file)) {
-		ast_log(LOG_WARNING, "Failed to create media for %s\n", uri);
+		ast_log(LOG_WARNING, "Failed to create media %d for %s\n", created, uri);
```

Comment:
I assume this is leftover debugging code of some kind?

Reply:
oops, yeah.  removed.